<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>npc_edit</class>
 <widget class="QWidget" name="npc_edit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>543</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="horizontalWidget_2" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <item row="0" column="1">
       <layout class="QGridLayout" name="gridLayout_11">
        <property name="bottomMargin">
         <number>20</number>
        </property>
        <item row="5" column="0" colspan="2">
         <widget class="QLabel" name="label_60">
          <property name="text">
           <string>Graphic Width: </string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QSpinBox" name="npc_edit_tab_graphicheight">
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
          <property name="value">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QSpinBox" name="npc_edit_tab_NpcHP">
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_61">
          <property name="text">
           <string>Graphic Height:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QComboBox" name="npc_edit_tab_graphiccombo"/>
        </item>
        <item row="5" column="2">
         <widget class="QSpinBox" name="npc_edit_tab_graphicwidth">
          <property name="minimum">
           <number>8</number>
          </property>
          <property name="maximum">
           <number>128</number>
          </property>
          <property name="value">
           <number>16</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_62">
          <property name="text">
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_63">
          <property name="text">
           <string>Graphic:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_64">
          <property name="text">
           <string>Hit Points:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QCheckBox" name="npc_edit_tab_canshoot">
          <property name="text">
           <string>Can Shoot</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_65">
          <property name="text">
           <string>*IA Type:</string>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QComboBox" name="npc_edit_tab_iatype"/>
        </item>
        <item row="8" column="2">
         <widget class="QComboBox" name="npc_edit_tab_shieldtype">
          <item>
           <property name="text">
            <string>no shield</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>full shield</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>front shield</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>stand_shield</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_66">
          <property name="text">
           <string>Shield:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_67">
          <property name="text">
           <string>Speed:</string>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <spacer name="verticalSpacer_9">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label_68">
          <property name="text">
           <string>Range:</string>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QSpinBox" name="npc_edit_tab_movespeed">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>8</number>
          </property>
          <property name="value">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QSpinBox" name="npc_edit_tab_range">
          <property name="minimum">
           <number>48</number>
          </property>
          <property name="maximum">
           <number>320</number>
          </property>
          <property name="value">
           <number>106</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_70">
          <property name="text">
           <string>Npc:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="npc_edit_tab_selectnpccombo"/>
        </item>
        <item row="13" column="0">
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>Add New</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QLineEdit" name="npc_edit_tab_NpcName">
          <property name="text">
           <string>My NPC</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="2">
       <widget class="QWidget" name="verticalWidget_2" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_10">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QWidget" name="verticalWidget_3" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QScrollArea" name="scrollArea_6">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="horizontalScrollBarPolicy">
               <enum>Qt::ScrollBarAlwaysOn</enum>
              </property>
              <property name="widgetResizable">
               <bool>false</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents_7">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>434</width>
                 <height>558</height>
                </rect>
               </property>
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>100</horstretch>
                 <verstretch>100</verstretch>
                </sizepolicy>
               </property>
               <property name="autoFillBackground">
                <bool>false</bool>
               </property>
               <widget class="npcPreviewArea" name="npc_edit_tab_previewarea" native="true">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>651</width>
                  <height>581</height>
                 </rect>
                </property>
                <property name="styleSheet">
                 <string notr="true"/>
                </property>
               </widget>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_19">
           <item>
            <widget class="QLabel" name="label_69">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Duration:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="npc_edit_tab_frameduration">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>9999</number>
             </property>
             <property name="value">
              <number>20</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QComboBox" name="npc_edit_tab_frametype">
           <item>
            <property name="text">
             <string>NONE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_STAND</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_WALK</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_JUMP</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_ATTACK</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_TELEPORT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_JUMP_ATTACK</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_WALK_ATTACK</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_HIT</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_STAIRS</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_STAIRS_SEMI</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_STAIRS_ATTACK</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_STAIRS_MOVE</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_TROW</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_TURN</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_MOVE_SEMI</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_WALK_AIR</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_INTRO</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>ANIM_SPECIAL_ATTACK</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>npcPreviewArea</class>
   <extends>QWidget</extends>
   <header>npcpreviewarea.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
